src/components/SimpleFeedback.js

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { css } from '@emotion/core';
+import { Button, Icon, PageTools } from '@newrelic/gatsby-theme-newrelic';
+
+const SimpleFeedback = ({ positiveUrl, negativeUrl }) => (
+  <PageTools.Section>
+    <PageTools.Title>Feedback</PageTools.Title>
+    <div
+      css={css`
+        font-size: 0.875rem;
+      `}
+    >
+      Was this page helpful?
+    </div>
+    <div
+      css={css`
+        display: flex;
+        margin-top: 0.5rem;
+        justify-content: center;
+        gap: 0.5rem;
```

Comment:
It looks like maybe Safari [doesn't have support](https://caniuse.com/?search=gap) for `gap`. Do we care about that? Should we have a fallback for that? Thoughts?

In case we need it, we can use `@supports`:

```sass
@supports not (gap: 0.5rem) {
  > *:first-child {
    margin-right: 0.5rem;
  }
}
```

Reply:
Ugh, this is what happens when I switch to a chromium-based browser. Thanks for catching that...I'll use this suggestion.